fecgov/openFEC
# Feature/4109 swagger is not in package file

## Summary

- Resolves #4109 
- Moved the Node package swagger-tools from the global scope into the package file.
- Updated config file to no longer install swagger-tools globally but to reference package.json
- Updated documentation around the simpler swagger-tools installation
- Added "name", "version", "license", and "repository" entries to the package file to soothe npm warnings (copied license type from fec-cms)
- Re-ordered the existing sections in package.json to be more human-friendly (meta, scripts, dependencies, devDependencies, engines)

## How to test the changes locally
- Pull the branch as usual
- `npm un -g swagger-tools` (removing swagger-tools from the global scope)
- `npm i` (`npm i` = `npm install`, will install everything in package.json and create package-lock.json)
- `./manage.py runserver`
- `npm run build`
- The build should go smoothly
- The Swagger page should work as it normally does

## Impacted areas of the application
Only the build and/or JavaScript should be affected, and only where Swagger is concerned

## Related PRs
None
